test/with_dummyserver/test_socketlevel.py

```diff
@@ -837,9 +856,9 @@ def socket_handler(listener):
                        ':9A:8C:B6:07:CA:58:EE:74:5E')
 
         def request():
+            pool = HTTPSConnectionPool(self.host, self.port,
```

Comment:
Why is there so much code re-organization in this pull request? It doesn't belong.